frontend/components/AuctionEventsBlock.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    <div class="mt-5">
+        <p class="mb-2">
+            The auction was taken via {{ takeEvents.length }} transaction<span v-if="takeEvents.length !== 1">s</span>
+            at {{ takeEvents[takeEvents.length - 1].transactionDate.toISOString() }}.
+        </p>
+        <ul class="list-disc list-inside">
+            <li v-for="(event, index) of takeEvents" :key="index">
+                Transaction <FormatAddress :value="event.transactionHash" :shorten="true" />
+                <span v-if="event.transactionDate"> executed <TimeTill :date="event.transactionDate" /> </span>
+            </li>
+        </ul>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { TakeEvent } from 'auctions-core/dist/src/types';
+import FormatAddress from '~/components/utils/FormatAddress.vue';
+import TimeTill from '~/components/common/TimeTill.vue';
+
+export default Vue.extend({
+    name: 'AuctionEventsBlock',
+    components: { FormatAddress, TimeTill },
+    props: {
+        takeEvents: {
+            type: Array as Vue.PropType<TakeEvent[]>,
+            default: null,
```

Comment:
If the default is null, you need to add checks to the template, since `null.length` will throw
```suggestion
            required: true,
```